www/src/components/Affirmations/Affirmations.tsx

```diff
@@ -1,6 +1,7 @@
 import { gql, useMutation, useQuery, useSubscription } from "@apollo/client";
 import { User } from "@prisma/client";
-import { message } from "antd";
+import React from 'react';
+import { snackBarContext } from "../../context/AlertContext/SnackBarProvider";
```

Comment:
If you look at the AuthContext.tsx file, you'll see how we can export a hook which would clean up our usage a bit.

Would change this to something like `import { useSnacks } from "../../context/AlertContext/SnackBarProvider";`